bin/dump_data.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Dump component data
+# Run this script from the root of the repository
+# dump_data.sh for all apps or dump_data.sh zaken documenten ... to specify specific apps
+
+set -e
+
+DEFAULT_APPS=(besluiten catalogi documenten zaken)
+
+export PGHOST=${DB_HOST:-db}
+export PGPORT=${DB_PORT:-5432}
+export PGUSER=${DB_USER:-postgres}
+export PGDATABASE=${DB_NAME:-postgres}
+export PGPASSWORD=${DB_PASSWORD:-""}
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+
+${SCRIPTPATH}/wait_for_db.sh
+
+DUMP_FILE=${DUMP_FILE:-"dump_$(date +'%Y-%m-%d_%H-%M-%S').sql"}
+
+# export given apps or export DEFAULT_APPS
+if [ "$#" -gt 0 ]; then
+  APPS=("$@")
+else
+  APPS=("${DEFAULT_APPS[@]}")
+fi
+
+>&2 echo "exporting: ${APPS[*]}"
+
+# create -t flags for each app
+INCLUDES=()
+for app in "${APPS[@]}"; do
+    INCLUDES+=("-t" "${app}_*")
+done
+
+pg_dump "${INCLUDES[@]}" -f "$DUMP_FILE"
```

Comment:
Not using `--data-only` here does mean that you can't import this data into another instance that already has its tables set up (migrations have run), because it will also export all the table info, indexes etc.

@joeribekker I'm not 100% if the use case of this script is to dump data and load it somewhere else, but if that is the case, I think it should be dumping data only.

Reply:
Changed it to do the following:

By default it will create a complete schema dump and a separate data dump for all apps, The datadump will disable triggers because zgw_consumers.Service objects won't exist.

with `--data-only`, `--schema-only` only one sql dump will be created

using `--combined` will append the data dump to the schema dump